packages/rn-tester/android/app/src/main/java/com/facebook/react/uiapp/RNTesterApplication.kt

```diff
@@ -0,0 +1,161 @@
+                          true // isTurboModule
+                        ),
+                      SampleLegacyModule.NAME to
+                        ReactModuleInfo(
+                          SampleLegacyModule.NAME,
+                          "SampleLegacyModule",
+                          false,  // canOverrideExistingModule
+                          false,  // needsEagerInit
+                          false,  // isCxxModule
+                          false // isTurboModule
+                        )
+                    )
+                  } else {
+                    emptyMap()
+                  }
+                }
+
+              },
+              object : ReactPackage {
+                override fun createNativeModules(
+                  reactContext: ReactApplicationContext): List<NativeModule> {
+                  return emptyList()
+                }
+
+                override fun createViewManagers(
+                  reactContext: ReactApplicationContext
+                ): List<ViewManager<*, *>> =
+                  listOf(MyNativeViewManager(), MyLegacyViewManager(reactContext));
+              })
+          }
+
+          override val isNewArchEnabled: Boolean = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED
+          override val isHermesEnabled: Boolean = BuildConfig.IS_HERMES_ENABLED_IN_FLAVOR
+        }
+      }
+
+
+    override fun onCreate() {
+        ReactFontManager.getInstance().addCustomFont(this, "Rubik", R.font.rubik)
+        super.onCreate()
+        SoLoader.init(this,  /* native exopackage */false)
+        if (BuildConfig.IS_NEW_ARCHITECTURE_ENABLED) {
+            load()
+        }
+        if (ReactFeatureFlags.enableBridgelessArchitecture) {
+            // TODO: initialize Flipper for Bridgeless
+        } else {
+            initializeFlipper(this, reactNativeHost.reactInstanceManager)
+        }
+    }
+
+    @UnstableReactNativeAPI
+    override val reactHost: ReactHost by lazy {
+        // Create an instance of ReactHost to manager the instance of ReactInstance,
+        // which is similar to how we use ReactNativeHost to manager instance of ReactInstanceManager
+        val reactHostDelegate = RNTesterReactHostDelegate(applicationContext)
+        val reactJsExceptionHandler = RNTesterReactJsExceptionHandler()
+        val componentFactory = ComponentFactory()
+        register(componentFactory)
+        val reactHostImpl = ReactHostImpl(
+                this.applicationContext,
+                reactHostDelegate,
+                componentFactory,
+                true,
+                reactJsExceptionHandler,
+                true)
+        if (BuildConfig.IS_HERMES_ENABLED_IN_FLAVOR) {
+          reactHostImpl.jsEngineResolutionAlgorithm = JSEngineResolutionAlgorithm.HERMES
+        } else {
+          reactHostImpl.jsEngineResolutionAlgorithm = JSEngineResolutionAlgorithm.JSC
+        }
+        reactHostDelegate.reactHost = reactHostImpl
+     reactHostImpl
```

Comment:
if you use `.apply{}` as suggested this line can also go

Reply:
Neat suggestion, that's way better. Done ✅